pmm-app/src/pmm-add-instance-app-panel/panel.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import AddRemoteInstance from './AddInstance/AddRemoteInstance/AddRemoteInstance';
+import DiscoveryPanel from './DiscoveryPanel/DiscoveryPanel';
+import '../react-plugins-deps/styles.scss';
+import '../react-plugins-deps/style.less';
+import AddInstance from './AddInstance/AddInstance';
+import { Button } from 'antd';
+import './panel.scss';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { createBrowserHistory } from 'history';
+
+const history = createBrowserHistory({});
+const AddInstancePanel = props => {
+  // @ts-ignore
+  const urlParams = new URLSearchParams(window.location.search);
+
+  const [selectedInstance, selectInstance] = useState({
+    type: urlParams.get('instance_type'),
+  });
+
+  const setSelectedInstance = instance => {
+    const url = new URL((window.location as unknown) as string);
+    url.searchParams.set('instance_type', instance.type);
+    selectInstance(instance);
```

Comment:
does `selectInstance` use `instance` at all? Looks like there is no reason to send `instance`

Reply:
Nope, It's being used, we adding it to selectedInstance